.gitignore

```diff
@@ -0,0 +1,6 @@
+# .gitignore
+node_modules/
+
+linters.yml
+
```

Comment:
- [ ] Kindly, check the logo of Microverse on the README file, since it's not appearing on it.

![image](https://user-images.githubusercontent.com/93630700/204561318-6db24788-79c6-4c01-bba8-d566ac2a8d41.png)

Reply:
Hi, thanks a lot for the review. 

The logo isn't showing because I didn't add the file to my folder. Can I remove the Logo or is it a must I add it?